src/Tween.js

```diff
@@ -99,235 +99,249 @@ else {
 }
 
 
+function assign(target, source) {
+	var keys = Object.keys(source);
+	var length = keys.length;
+
+	for (var i = 0; i < length; i += 1) {
+		target[keys[i]] = source[keys[i]];
+	}
+
+	return target;
+}
```

Comment:
@mikebolt Hmmm, I forgot why I did this in place of Object.assign. I don't usually do things just for the heck of it though. Next time I'll be sure to note reasons for things.

Reply:
Oh right, because older browsers don't have Object.assign. A transpile step can help here. There's also http://polyfill.io, which is really nice.